src/d3d12/d3d12_settings.hpp

```diff
@@ -48,6 +48,6 @@ namespace wr::d3d12::settings
 	static const constexpr bool enable_object_culling = true;
 	static const constexpr unsigned int num_max_rt_materials = 3000;
 	static const constexpr unsigned int num_max_rt_textures = 1000;
-	static const constexpr unsigned int fallback_ptrs_offset = 1013;
+	static const constexpr unsigned int fallback_ptrs_offset = 3500;
```

Comment:
with this change can this now be removed? everything now has its own heap. so no need to hard code the offset anymore.